e2e-tests/src/test/rewards.rs

```diff
@@ -40,6 +41,127 @@ fn get_non_reserved_members_for_session(config: &Config, session: SessionIndex)
     non_reserved.iter().map(account_from_keypair).collect()
 }
 
+fn validators_bond_extra_stakes(config: &Config, additional_stakes: Vec<u128>) {
+    let node = &config.node;
+    let root_connection = config.create_root_connection();
+
+    let validator_seeds = get_validators_seeds(config);
+    let controller_seeds: Vec<_> = validator_seeds
+        .iter()
+        .map(|v| format!("{}//Controller", v))
+        .collect();
+
+    let controller_accounts_key_pairs = accounts_seeds_to_keys(&controller_seeds);
+    let stash_accounts_key_pairs = accounts_seeds_to_keys(&validator_seeds);
+
+    let controller_accounts: Vec<AccountId> = controller_accounts_key_pairs
+        .iter()
+        .map(account_from_keypair)
+        .collect();
+
+    balances_batch_transfer(&root_connection.as_signed(), controller_accounts, TOKEN);
+
+    stash_accounts_key_pairs
+        .iter()
+        .zip(additional_stakes.iter())
+        .for_each(|(key, additional_stake)| {
+            let id = account_from_keypair(key);
+            balances_transfer(
+                &root_connection.as_signed(),
+                &id,
+                *additional_stake + TOKEN,
```

Comment:
Question: why `+TOKEN`? if this is for being over existential deposit, then I would either add comment or create some constant like `MIN_ACCOUNT_DEPOSIT = TOKEN` and use it here

Reply:
If I understand correctly, we need some additional funds to make sure that fees can be covered. I will add a comment.